pegasus-all/build.gradle

```diff
@@ -0,0 +1,12 @@
+/*
+    This is a meta-project that programmatically depends on all other consumer-facing modules
+    so that the entire dependency tree of pegasus may be pulled in easily. Be warned that this
+    module is not intended to be directly consumed, since it'll bloat the consumer's dependencies.
```

Comment:
Based on the wording here, it's usage is somewhat non-standard? I will recommend adding to somewhere to let consumer's know.

Reply:
I'm not too sure how we can surface this to people. I think I'll merge this for now and ponder that afterward.